pyproject.toml

```diff
@@ -7,9 +7,12 @@ requires = [
 build-backend = "setuptools.build_meta"
 
 [project]
+name = "minorminer"
+dynamic = ["version"]
 authors = [
-    {name = "Kelly Boothby", email = "boothby@dwavesys.com"},
```

Comment:
IMO we should so
```.toml
authors = [
 {name = "Kelly Boothby"},
]
maintainers = [
    {name = "D-Wave Inc.", email = "tools@dwavesys.com"},
]
```

Reply:
Happy to go with that; makes sense to me.